src/azul/service/manifest_service.py

```diff
@@ -103,6 +103,7 @@ class ManifestFormat(Enum):
     compact = 'compact'
     full = 'full'
     terra_bdbag = 'terra.bdbag'
+    bulk_download = 'bulk.download'
```

Comment:
```suggestion
    curl = 'curl'
```

Likewise s/BulkDownloadManifest/CurlManifest/g.